cupy/fallback_mode/data_tranfer.py

```diff
@@ -0,0 +1,59 @@
+"""
+Data transfer methods
+"""
+
+import cupy as cp
+import numpy as np
+
+
+def vram2ram(args, kwargs):
+    """
+    Transfers ndarrays in *args, **kwargs from GPU to CPU.
+
+    Args:
+        args (tuple): Arguments.
+        kwargs (dict): Keyword arguments.
+
+    Returns:
+        cpu_args (tuple): Arguments in CPU.
+        cpu_kwargs (dict): Keyword arguments in CPU.
+    """
+    cpu_args = []
+    cpu_kwargs = {}
+
+    for arg in args:
+        if isinstance(arg, cp.ndarray):
+            cpu_args.append(cp.asnumpy(arg))
```

Comment:
Code duplications. How about writing as follows?
```py
def _get_xp_args(orig_module, to_xp, arg):
    if isinstance(arg, orig_module.ndarray):
        return to_xp(arg)
    if isinstance(arg, tuple):
        return tuple([_get_xp_args(x) for x in arg])
    if isinstance(arg, dict):
        return dict([key, _get_xp_args(x) for x in arg.items()])
    if isinstance(arg, (bool, int, float, numpy.generic)) or callable(arg):
        return arg
    ...
    raise NotImplementedError

def _get_numpy_args(args, kwargs):
    return _get_xp_args(cupy, cupy.asnumpy, (args, kwargs))

def _get_cupy_args(args, kwargs):
    return _get_xp_args(numpy, cupy.array, (args, kwargs)
```

Reply:
Yes, this definitely looks better. What about `list`?